### PR TITLE
Add unofficial support for Apple Silicon, see: #168

### PR DESCRIPTION
**Pull request by k4rtik**

### CODE REVIEW

File: opam/opam-coq-archive/extra-dev/packages/gappa/gappa.dev/opam [hunk 1]
```diff
@@ -14,6 +14,8 @@ build: [
   [ "./configure"
     # If someone knows how to ask MacPorts for the usual include and lib path, please tell me
     "CXXFLAGS=-I/opt/local/include" "LDFLAGS=-L/opt/local/lib" { os-distribution = "macports" & os = "macos" }
+    # Support installing on Apple Silicon with Homebrew
+    "CXXFLAGS=-I/opt/homebrew/include" "LDFLAGS=-L/opt/homebrew/lib" { os-distribution = "homebrew" & os = "macos" }
```
Comment (MSoegtropIMC) [inline opam/opam-coq-archive/extra-dev/packages/gappa/gappa.dev/opam:18]: I wonder if `/opt/homebrew` is now the default destination for homebrew - it used to be `/opt/local`.

**Review by MSoegtropIMC — COMMENTED** [review]
The homebrew default path question needs clarification. CI uses homebrew and it works on Mac and I don't think the default path depends on the architecture - this might be something specific to your system.

File: opam/opam-coq-archive/extra-dev/packages/gappa/gappa.dev/opam (hunk 1 above)
Comment (k4rtik) [reply opam/opam-coq-archive/extra-dev/packages/gappa/gappa.dev/opam:18]: It is the default on Apple Silicon, see https://docs.brew.sh/Installation

**Review by MSoegtropIMC — APPROVED** [review]
LGTM

**Pull request merged**